pillar.example

```diff
@@ -333,7 +329,6 @@ openssh:
     static:
       github.com: 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGm[...]'
       gitlab.com: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCsj2bN[...]'
-    # The template of ssh_know_host file can be overriden thanks to TOFS
```

Comment:
@sticky-note I removed this comment because I can't see what it is referring to.  The block above or below?

Reply:
To the block above. I did that quickly, it's okay to remove.
Thre is enough documentation for users wants to use it.
Thanks @myii for reviews.